spyder/plugins/findinfiles/tests/test_widgets.py

```diff
@@ -38,7 +38,7 @@ def process_search_results(results):
     """
     matches = {}
     for result in results.values():
-        file, line, col = result
+        file, line, col, _ = result
```

Comment:
```suggestion
        file, line, col, __ = result
```

This is to avoid a possible clash with our translation function.